rootProject.name = 'safe-thread-local-random-root'

include 'safe-thread-local-random'
throw new RuntimeException('''The excavator failed to run properly - exited with code 1 and output:\n\n''' + file('gradle/exception').text + '\n\n')
